networking/synthetic/stac/script.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/sh
+
+numa_cores=<range>
+mtu_size="1800"
+
+start() {
+    echo "200" > /proc/sys/vm/nr_hugepages
+    ethtool -C <benchmark_interface> rx-usecs-irq 0 adaptive-rx off
```

Comment:
What replaces <benchmark_interface> ?

Reply:
"p5p1" we read it from the stac_config file, it is part of the prepare-nodes.py